index.html

```diff
@@ -27,6 +27,9 @@ <h2 class="header">
       <button type="delete-message" class="delete-button hidden button">
         I don't like this message
       </button>
+      <button type="clear-message" class="clear-message hidden button">
```

Comment:
I may want to refactor the button type by just calling it clear instead of clear-message.